.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
I would not commit vscode config here

Reply:
Fudge - mistake from my part. Managed to remember to remove it in every commit until now. My bad!

I'll add the .vscode-folder to gitignore :)

Reply:
Maybe ignore it globally?

Reply:
Did not know I could have a global gitignore. Now that's set up :)